package.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "opentoken-lib",
+    "version": "0.0.1",
```

Comment:
The most recent I projects I've code reviewed have all started at 1.0.0, but I'm not sure that that's necessarily correct either. Perhaps something we should talk about as a team.